Makefile

```diff
@@ -30,7 +30,7 @@ format-check:
 	black . --check
 
 lint:
-	python3 -m prospector -X  --profile-path .prospector.yaml
+	prospector -X  --profile-path .prospector.yaml
```

Comment:
We are now executing our tasks under peynv managed environment therefore we don't need python executable in the command